docs/pages/docs/core-concepts/workspaces.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Workspaces
+description: Use workspaces to organize your turborepo apps and packages.
+---
+
+import Callout from "../../../components/callout";
+import HeartIcon from "@heroicons/react/solid/HeartIcon";
+import { Tabs, Tab } from '../../../components/Tabs'
+
+# Getting started with workspaces
+
+Workspaces are a feature implemented by package managers to assist with working on multiple apps and packages in the same codebase.
+
+Turborepo is compatible with three package managers, [npm](https://docs.npmjs.com/cli/v8/using-npm/workspaces#description), [Yarn](https://classic.yarnpkg.com/lang/en/docs/workspaces/), and [pnpm](https://pnpm.io/workspaces), each with its own implementation of workspaces and functionality. Your package manager will determine how your apps and packages are organized in your workspaces and run in your turborepo.
+
+## Configuring workspaces
+
+To use workspaces, you'll first need to declare their file system locations to your package manager
+
+A convention is to have an `apps` folder and a `packages` folder. The `apps` folder can contain workspaces for launchable apps, such as a Next.js or Svelte app. The `packages` folder can contain workspaces for packages that are used by either an app, or another package.
```

Comment:
"A convention" sounds a bit odd.

Can we change this to "A common convention" or perhaps" The recommended convention"?

Reply:
```suggestion
A common convention is to have an `apps` folder and a `packages` folder. The `apps` folder can contain workspaces for launchable apps, such as a Next.js or Svelte app. The `packages` folder can contain workspaces for packages that are used by either an app, or another package. This isn't a requirement, just a suggestion we've found useful.
```